src/org/labkey/test/util/TestDataGenerator.java

```diff
@@ -343,13 +343,18 @@ public static String randomInvalidDomainName(int size)
     }
 
     public static String randomDomainName(int size)
+    {
+        return randomDomainName(size, null);
+    }
+
+    public static String randomDomainName(int size, @Nullable String prefix)
```

Comment:
Maybe add a comment about the difference in behavior between a null prefix and a not null prefix.